examples/rag_evals_via_nvidia_chatrag_bench.py

```diff
@@ -108,26 +108,26 @@ def __call__(self, input: dict) -> CandidateResponse:  # noqa: A002
             },
         )
 
-
 candidates = [CustomRAGAgentCandidate(**params) for params in param_combinations]
-harness = EvalHarness(
-    num_cpus=None,
-    async_batch_size=1,
-    num_samples=1,
-)
-harness.add_evals_from_yamls("examples/nvidia_chatrag_bench/evals/*.yaml")
+harness = EvalHarness(generate_mode=Mode.PARALLEL, eval_mode=Mode.PARALLEL)
+harness.add_evals_from_files("examples/nvidia_chatrag_bench/evals/*.yaml")
 harness.add_candidates(candidates)
 
-
-def save_callback(result: EvalResult) -> None:
-    """Simple callback function to print progress and save the eval result."""
-    timestamp = time.strftime('%Y-%m-%d %H:%M:%S')
-    print(f"{timestamp}: {result.candidate_obj.metadata['id']} - {result.eval_obj.metadata['uuid']}")  # noqa: E501
-    # save result
-    result.to_yaml(f"examples/nvidia_chatrag_bench/results/{result.eval_obj.metadata['uuid']}-{result.candidate_obj.metadata['uuid']}.yaml")
-
-
-harness.callback = save_callback
 print(f"Starting Evals: {len(harness.evals)} evals and {len(harness.candidates)} candidates (total {len(harness.evals) * len(harness.candidates)} eval-candidate pairs)")  # noqa: E501
-results = harness()
-print("Finished Evals")
+print(f"Timestamp: {time.strftime('%Y-%m-%d %H:%M:%S')}")
+results: list[CandidateRunResult] = harness()
+print(f"Finihsed - Timestamp: {time.strftime('%Y-%m-%d %H:%M:%S')}")
```

Comment:
```suggestion
print(f"Finished - Timestamp: {time.strftime('%Y-%m-%d %H:%M:%S')}")
```

Also, maybe in the future, incorporating logging through like logaru?